examples/llm/mcp.ts

```diff
@@ -0,0 +1,12 @@
+// Create instance
+const llm = BlessLLM("Llama-3.1-8B-Instruct-q4f16_1-MLC");
+
+// Set options
+llm.setOptions({
+  tools_sse_urls: [
+    "http://localhost:3001/sse",
+  ],
+});
+
+// Chat
+console.log(llm.chat("Add the following numbers: 1215, 2213"));
```

Comment:
is it possible to change initial prompt?

Reply:
maybe in future we can add more examples - which include hosting an MCP server too (a basic one) - or maybe even pointing to a public one.